AssertionError: Expected filename to be string, passed in storage/2/1/21c318079d0af8b5e9f91e0181a1d618

https://sentry.io/learningequality/studio/issues/686238721/

```
AssertionError: Expected filename to be string, passed in storage/2/1/21c318079d0af8b5e9f91e0181a1d618
(17 additional frame(s) were not displayed)
...
  File "rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "rest_framework_bulk/drf3/mixins.py", line 74, in bulk_update
    self.perform_bulk_update(serializer)
  File "rest_framework_bulk/drf3/mixins.py", line 85, in perform_bulk_update
    return self.perform_update(serializer)
  File "rest_framework_bulk/drf3/mixins.py", line 82, in perform_update
    serializer.save()
  File "rest_framework/serializers.py", line 654, in save
    self.instance = self.update(self.instance, validated_data)

AssertionError: Expected filename to be string, passed in storage/2/1/21c318079d0af8b5e9f91e0181a1d618
```

--- a/contentcuration/contentcuration/utils/gcs_storage.py
+++ b/contentcuration/contentcuration/utils/gcs_storage.py
@@ -28,8 +28,6 @@ class GoogleCloudStorage(Storage):
         Raises an AssertionError if filename is not a string.
         """
 
-        assert isinstance(filename, str), "Expected filename to be string, passed in {}".format(filename)
-
         typ, _ =  mimetypes.guess_type(filename)
 
         if not typ:

